aspnetcore/blazor/components/templated-components.md

```diff
@@ -20,49 +22,50 @@ Templated components are components that receive one or more UI templates as par
 
```

Comment:
On the preceding line, let's drop "couple" ... and it won't let me make a formal GH suggestion, so I'll place the last sentence in a DIFF ...

```diff
- A couple of examples include:
+ Examples include:
```